docs/blog/2018-10-11-rise-of-modern-web-development/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Journey to the Content Mesh Part 3: The Rise of Modern Web Development"
+author: "Sam Bhagwat"
+"date": 2018-10-09
+tags: ["content-mesh"]
+---
+
+_This is Part 3 of a series. Part 1 is_ [The Journey to a Content Mesh](/blog/2018-10-04-journey-to-the-content-mesh) _and Part 2 is_ [Unbundling of the CMS](/blog/2018-10-10-unbundling-of-the-cms).
+
+Developing for the web is _hard_. With millions of properties on the Internet, website developers have to create experiences that are compelling enough to engage audiences when new destinations are only a few keystrokes away.
+
+And while creating rich user experiences is never an easy task, standard content web development environments introduce additional challenges. Teams are forced to work around them, but ultimately deliver less rich experiences at higher cost.
```

Comment:
> standard content web development environments

I find this wording a little awkward. "standard content management systems"? "managing content within a web development environment"?

Reply:
Thanks. I'm trying to say "walled-garden CMS development environments" but apparently didn't quite get there :)